### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by yifanplanet**
# Code changes

- [x] Fixed and disabled several failing tests
- [x] Frozen the time for availability and scheduler tests

Several tests had to be disabled as part of this PR because they kept failing with Github Actions. These have been documented in the story #76731

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by AaronDDM — APPROVED** [review]

**Review by krisgardiner — APPROVED** [review]

File: components/composer/src/Composer.svelte [hunk 1]
```diff
@@ -790,10 +790,10 @@
       </main>
       <footer>
         {#if _this.show_attachment_button && (id || uploadFile)}
-          <label for="file-upload" class="composer-btn file-upload">
-            <AttachmentIcon class="AttachmentIcon" />
+          <button for="file-upload" class="composer-btn file-upload">
```
Comment (ozsivanov) [inline components/composer/src/Composer.svelte:793]: Why was this changed? If we're keeping this as a button, `for="file-upload"` should be removed since it's a label prop.

**Review by ozsivanov — APPROVED** [review]
